packaging/nntrainer.spec

```diff
@@ -106,11 +110,19 @@ Requires:	nnstreamer-tensorflow2-lite
 %endif # support_nnstreamer_backbone
 
 %if 0%{?support_tflite_backbone}
+%if 0%{tizen_version_major}%{tizen_version_minor} <= 60
+BuildRequires: tensorflow-lite-devel
+%else
 BuildRequires: tensorflow2-lite-devel
+%endif
 %endif # support_tflite_backbone
 
 %if 0%{?support_tflite_interpreter}
+%if 0%{tizen_version_major}%{tizen_version_minor} <= 60
+BuildRequires: tensorflow-lite-devel
+%else
 BuildRequires: tensorflow2-lite-devel
+%endif
 %endif # support_tflite_interpreter
```

Comment:
Create a macro for Tizen < 6.5 (== Tizen <= 6.0).

```
%if 0%{?tizen_version_major}%{?tizen_version_minor} < 65
%define tizen65plus 0
%else
%define tizen65plus  1
%endif
```

Merge line 112-126.

```
%if 0%{?support_tflite_backbone} || 0%{?support_tflite_interpreter}
%if 0%{?tizen65plus}
BuildRequires: tensorflow2-lite-devel
%else
BuildRequires: tensorflow-lite-devel
%endif
%endif
```

Reply:
this will be removed after tensorflow2 spec in tizen 6.0 is fixed.